lua/lspconfig/dartls.lua

```diff
@@ -5,7 +5,10 @@ local server_name = 'dartls'
 local bin_name = 'dart'
 
 local find_dart_sdk_root_path = function()
-  if vim.fn['executable'] 'flutter' == 1 then
+  if os.getenv 'FLUTTER_SDK' ~= nil then
+    local flutter_path = os.getenv 'FLUTTER_SDK'
+    return util.path.path_join(flutter_path, '/cache/dart-sdk/bin/dart')
```

Comment:
Use `vim.env.FLUTTER_SDK` instead of `os.getenv('FLUTTER_SDK')`.

Reply:
why?

Reply:
Because they're already defined, while `os.getenv` needs to do a lookup.

Reply:
It's basically free, I would generally prefer avoiding calling vimscript functions if possible.